$OpenBSD: patch-Source_ThirdParty_ANGLE_src_compiler_translator_glslang_y,v 1.2 2015/09/23 06:06:11 ajacoutot Exp $

XXX rm when bison(1) is updated
Source/ThirdParty/ANGLE/src/compiler/glslang.y:52.1-5: invalid directive: `%code'

--- Source/ThirdParty/ANGLE/src/compiler/translator/glslang.y.orig	Mon Sep 14 09:08:10 2015
+++ Source/ThirdParty/ANGLE/src/compiler/translator/glslang.y	Mon Sep 21 14:54:56 2015
@@ -50,12 +50,10 @@ WHICH GENERATES THE GLSL ES PARSER (glslang_tab.cpp AN
 %define api.pure full
 %locations
 
-%code requires {
+%union {
 #define YYLTYPE TSourceLoc
 #define YYLTYPE_IS_DECLARED 1
-}
 
-%union {
     struct {
         union {
             TString *string;
